automation_tools/satellite6/capsule.py

```diff
@@ -206,7 +207,11 @@ def generate_capsule_certs(capsule_hostname, force=False):
         fqdn_opt = '--foreman-proxy-fqdn'
     else:
         fqdn_opt = '--capsule-fqdn'
-    cert_path = '{0}-certs.tar'.format(capsule_hostname)
+    if bz_bug_is_open(1466688):
+        # Absolute path bug
+        cert_path = '~/{0}-certs.tar'.format(capsule_hostname)
```

Comment:
this may not work because of another bug: 1470653

Reply:
We are not double quoting the cert_path string so it should work

Reply:
Looks like automation won't include quotes.. in that case it will work.